<?xml version="1.0"?>
<sdf version="1.9">
    <model name="rgbd_camera">
        <link name="rgbd_camera">
            <inertial>
                <mass>0.005</mass>
                <inertia>
                    <ixx>8.33e-06</ixx>
                    <ixy>0</ixy>
                    <ixz>0</ixz>
                    <iyy>8.33e-06</iyy>
                    <iyz>0</iyz>
                    <izz>8.33e-06</izz>
                </inertia>
            </inertial>
            <sensor name="rgbd_camera" type="rgbd_camera">
                <always_on>1</always_on>
                <update_rate>20</update_rate>
                <camera name="rgbd_camera">
                    <horizontal_fov>1.0472</horizontal_fov>
                    <lens>
                        <intrinsics>
                            <!-- fx = fy = width / ( 2 * tan (hfov / 2 ) ) -->
                            <fx>554.3</fx>
                            <fy>554.3</fy>
                            <!-- cx = ( width + 1 ) / 2 -->
                            <cx>320.5</cx>
                            <!-- cy = ( height + 1 ) / 2 -->
                            <cy>240.5</cy>
                            <s>0</s>
                        </intrinsics>
                    </lens>
                    <image>
                        <width>640</width>
                        <height>480</height>
                        <format>R8G8B8</format>
                    </image>
                    <clip>
                        <near>0.01</near>
                        <far>300</far>
                    </clip>
                    <depth_camera>
                        <clip>
                            <near>0.1</near>
                            <far>10</far>
                        </clip>
                    </depth_camera>
                    <noise>
                        <type>gaussian</type>
                        <mean>0</mean>
                        <stddev>0.007</stddev>
                    </noise>
              </camera>
            </sensor>
        </link>
        <frame name="mount_point"/>
    </model>
</sdf>
